analysis_options.yaml

```diff
@@ -7,8 +7,8 @@
 
 # The following lines activate a set of recommended lints.
 include:
-  - package:lint/strict.yaml
   - package:flutter_lints/flutter.yaml
+  - package:lint/strict.yaml
```

Comment:
Do we really need to keep both lint packages?

Reply:
It depends on what we want. 

What do you think about these three rules : library_private_types_in_public_api, unintended_html_in_doc_comment, use_key_in_widget_constructors ? Should we add them ? If not I guess that we can keep only the lint package.

Reply:
use_key_in_widget_constructors we definitely should have it, and library_private_types_in_public_api looks like a good practice too.

the third one we don't really care.

We should pick the set of rules that best fit our needs, and then adapt by manually enabling/disabling rule individually. Flutter lints is certainly a good choice so we can keep it. Have you found all the differences between flutter_lints and lint/strict?

Reply:
Yes there is google sheets at the end of [this article](https://rydmike.com/blog_flutter_linting.html) that show the differences but this is probably not up to date.

I'll disable the rule about html doc comment.